<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewFamilyEditorForm</class>
 <widget class="QDialog" name="NewFamilyEditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="motherGroupBox">
       <property name="title">
        <string>Mother / parent 1</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="motherButtonBar">
          <item>
           <widget class="QPushButton" name="motherNewPerson">
            <property name="text">
             <string>Add new</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::ListAdd"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="motherExistingPerson">
            <property name="text">
             <string>Select existing</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::InsertLink"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="motherForm">
          <item row="0" column="0">
           <widget class="QLabel" name="motherIdLabel">
            <property name="text">
             <string>ID</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="motherId">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="motherNameLabel">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="motherName">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="motherBirthLabel">
            <property name="text">
             <string>Birth</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="motherBirth">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="motherRelationLabel">
            <property name="text">
             <string>Relation</string>
            </property>
            <property name="buddy">
             <cstring>motherRelation</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="motherRelation"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="fatherGroupBox">
       <property name="title">
        <string>Father / relation 2</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="fatherButtonBar">
          <item>
           <widget class="QPushButton" name="fatherNewPerson">
            <property name="text">
             <string>Add new</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::ListAdd"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="fatherExistingPerson">
            <property name="text">
             <string>Select existing</string>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::InsertLink"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="fatherForm">
          <item row="3" column="0">
           <widget class="QLabel" name="fatherRelationLabel">
            <property name="text">
             <string>Relation</string>
            </property>
            <property name="buddy">
             <cstring>fatherRelation</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QComboBox" name="fatherRelation"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="fatherBirthLabel">
            <property name="text">
             <string>Birth</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="fatherBirth">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="fatherNameLabel">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="fatherName">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="fatherIdLabel">
            <property name="text">
             <string>ID</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="fatherId">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="parentRelationGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Parent relationship</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="parentRelationRole"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Relationship between the parents. When two people have children togheter, they always need a relationship type, even if that type is none.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewFamilyEditorForm</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewFamilyEditorForm</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
